### DISCUSSION TITLE
💪 Answer this discussion to get Galaxy Brain 💪

### DISCUSSION THREAD

**Discussion by drknzz** [drknzz/GitHub-Achievements#48] [Q&A] [answered]
💪

**Comment by MitkoVtori** [comment, marked as answer]
Answering, Thanks

**Comment by S1i** [comment]
👌

**Comment by caaiopereira** [comment]
I am answering, Thanks!

**Comment by philipp-bliznuk** [comment]
Answering, Thanks

**Comment by AshNumpy** [comment]
Answering, Thanks!

**Comment by Xaypanya** [comment]
Answering, Thanks!

**Comment by 0xpako** [comment]
I like this idea :)

**Comment by barrientossjoel** [comment]
So useful! 👌

**Comment by StefanShivarov** [comment]
Here's my answer! Need this badge man! :D

**Comment by leodisarli** [comment]
Thnak you guys!

**Comment by priyamaggarwal18** [comment]
Nice to see how open source contributors help themselves.

Can you upvote this please :)

**Comment by Jikaao** [comment]
Answering, Thanks!

**Comment by the1Riddle** [comment]
am well and answering.

**Comment by Anto-Napo** [comment]
Guess I'll answer, thanks 👍

**Comment by bettyc0der** [comment]
I am answering and can you also star my repository thanks Im a student
I starred your repo please star mine
https://github.com/salcedocharlotte/CIS261-Course-Project
